### PR TITLE
Triton serving runtime

### PR DESCRIPTION
**Pull request by erwangranger**
a quick first pass at showing how to add custom serving runtimes, in this case, with triton. 

Let me know what you think, @guimou .

### CODE REVIEW

File: docs/odh-rhods/custom-runtime-triton.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
+# Deploying and using a Custom Serving Runtime in RHODS
+
```
Comment (guimou) [inline docs/odh-rhods/custom-runtime-triton.md:2]: I would modify the title to RHODS/ODH and add a quick note that the procedure, although described for RHODS, works the same for both.

File: docs/odh-rhods/custom-runtime-triton.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+## Before you start
+
+This document will guide you through the broad steps necessary to deploy a custom Serving Runtime to serve a model using the triton Runtime.
+
```
Comment (guimou) [inline docs/odh-rhods/custom-runtime-triton.md:6]: Capital T on Triton. I would also put the real name as a reference:
... using the Triton Runtime (NVIDIA Triton Inference Server).

**Review by guimou — COMMENTED** [review]
A few notes only, then good to merge!

File: docs/odh-rhods/custom-runtime-triton.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+While RHODS supports your ability to add your own runtime, it does not support the runtimes themselves. Therefore, it is up to you to configure, adjust and maintain the runtime itself.
+
+This document expects a bit of familiarity with RHODS.
+Click on **D
```
[the remark this answers is not in the document]
Comment (erwangranger) [reply docs/odh-rhods/custom-runtime-triton.md]: indeed.

File: docs/odh-rhods/custom-runtime-triton.md (hunk 1 above)
Comment (erwangranger) [reply docs/odh-rhods/custom-runtime-triton.md:4]: done in https://github.com/opendatahub-io-contrib/ai-on-openshift/pull/38/commits/0a759ddaa4597ddf7da23a67fd19537a4e4acf60

File: docs/odh-rhods/custom-runtime-triton.md (hunk 2 above)
Comment (erwangranger) [reply docs/odh-rhods/custom-runtime-triton.md:8]: done in https://github.com/opendatahub-io-contrib/ai-on-openshift/pull/38/commits/0a759ddaa4597ddf7da23a67fd19537a4e4acf60

**Comment by erwangranger** [comment]
Thanks for the feedback.

**Pull request merged**